digital_land/collect.py

```diff
@@ -77,15 +77,7 @@ def get(self, url, log={}, verify_ssl=True, plugin="get"):
                 timeout=120,
                 verify=verify_ssl,
             )
-        except (
-            requests.exceptions.SSLError,
-            requests.ConnectionError,
-            requests.HTTPError,
-            requests.Timeout,
-            requests.TooManyRedirects,
-            requests.exceptions.MissingSchema,
-            requests.exceptions.ChunkedEncodingError,
-        ) as exception:
+        except (requests.RequestException,) as exception:
```

Comment:
Does this work for all of the current errors?

Reply:
requests.RequestException seems to be a base of all the others, and the tests seem prove that.